### PR TITLE
[REF] auto-fix conventions (and manual fixes)

### PR DESCRIPTION
**Pull request by mdietrichc2c**

### CODE REVIEW

File: __unported__/partner_address_ldap/address.py [hunk 1]
```diff
@@ -51,6 +51,7 @@
 
 
 class LdapConnMApper(object):
+
     """LdapConnMApper: push specific fields from the Terp
```
Comment (bwrsandman) [inline __unported__/partner_address_ldap/address.py:54]: Is the newline before a doctring a positive change to style?

File: __unported__/partner_address_ldap/address.py [hunk 2]
```diff
@@ -114,6 +115,7 @@ def get_connexion(self):
 
 
 class LDAPAddress(orm.Model):
+
```
Comment (bwrsandman) [inline __unported__/partner_address_ldap/address.py]: Is the newline before a doctring a positive change to style?

File: __unported__/partner_address_ldap/company.py [hunk 3]
```diff
@@ -36,6 +36,7 @@
 
 
 class Res_company(orm.Model):
+
     """Define ldap connexion parameters"""
```
Comment (bwrsandman) [inline __unported__/partner_address_ldap/company.py:39]: Is the newline before a doctring a positive change to style?

File: __unported__/partner_address_ldap/partner.py [hunk 4]
```diff
@@ -36,6 +36,7 @@
 
 
 class LdapPartner(orm.Model):
+
     """Ensure that when deleting a partner unlink function is called on all
     related addresses"""
```
Comment (bwrsandman) [inline __unported__/partner_address_ldap/partner.py:40]: Is the newline before a doctring a positive change to style?

File: base_contact/base_contact.py [hunk 5]
```diff
@@ -216,13 +216,13 @@ class ir_actions_window(orm.Model):
     _inherit = 'ir.actions.act_window'
 
     def read(
-            self, cr, user, ids, fields=None, context=None,
+            self, cr, user, ids, field_list=None, context=None,
```
Comment (bwrsandman) [inline base_contact/base_contact.py]: Is this change necessary for the pylint tests? This seems terribly confusing.

File: base_location/better_zip.py [hunk 6]
```diff
@@ -23,6 +23,7 @@
 
 
 class BetterZip(orm.Model):
+
     " City/locations completion object"
```
Comment (bwrsandman) [inline base_location/better_zip.py:26]: Is the newline before a doctring a positive change to style?

File: base_partner_merge/base_partner_merge.py [hunk 7]
```diff
@@ -81,6 +81,7 @@ class MergePartnerLine(orm.TransientModel):
 
 
 class MergePartnerAutomatic(orm.TransientModel):
+
     """
     The idea behind this wizard is to create a list of potential partners to
     merge. We use two objects, the first one is the wizard for the end-user.
```
Comment (bwrsandman) [inline base_partner_merge/base_partner_merge.py:86]: Is the newline before a doctring a positive change to style?

File: base_partner_merge/base_partner_merge.py [hunk 8]
```diff
@@ -361,8 +362,10 @@ def _merge(self, cr, uid, partner_ids, dst_partner=None, context=None):
 
         if dst_partner and dst_partner.id in partner_ids:
             src_partners = proxy.browse(cr, uid,
-                                        [id for id in partner_ids
-                                         if id != dst_partner.id],
+                                        [partner_id
+                                         for partner_id
+                                         in partner_ids
+                                         if partner_id != dst_partner.id],
```
Comment (bwrsandman) [inline base_partner_merge/base_partner_merge.py]: Just a suggestion:
Style would be better if the new line was at the opening parenthesis, left right side heavy code is just as bad style.

``` python
           src_partners = proxy.browse(
               cr, uid, [
                  partner_id for partner_id in partner_ids
                  if partner_id != dst_partner.id
               ], context=context)
```

Better even would be to not calculate the ids in the function signature

``` python
           src_ids = [
                  partner_id for partner_id in partner_ids
                  if partner_id != dst_partner.id
           ]
           src_partners = proxy.browse(cr, uid, src_ids, context=context)
```

Easier to read and to debug

File: base_partner_sequence/partner.py [hunk 9]
```diff
@@ -25,6 +25,7 @@
 
 
 class ResPartner(orm.Model):
+
```
Comment (bwrsandman) [inline base_partner_sequence/partner.py]: Is the newline before a doctring a positive change to style?

File: base_partner_merge/validate_email.py [hunk 10]
```diff
@@ -68,7 +68,6 @@ class ServerError(Exception):
 
 
 def validate_email(email, check_mx=False, verify=False):
-
```
Comment (bwrsandman) [inline base_partner_merge/validate_email.py:71]: Here there's a removal of a new line before a docstring, while there are additions everywhere else.

File: partner_firstname/partner.py [hunk 11]
```diff
@@ -22,6 +22,7 @@
 
 
 class ResPartner(orm.Model):
+
```
Comment (bwrsandman) [inline partner_firstname/partner.py]: Is the newline before a doctring a positive change to style?

File: passport/tests/test_passport.py [hunk 12]
```diff
@@ -113,6 +115,7 @@ def test_passport(self):
 
 
 class Test_passport_name_get(TransactionCase):
+
     """Test name_get"""
```
Comment (bwrsandman) [inline passport/tests/test_passport.py:116]: Is the newline before a doctring a positive change to style?

File: passport/tests/test_passport.py [hunk 13]
```diff
@@ -75,6 +76,7 @@ def test_passport(self):
 
 
 class Test_passport_bad(Base_Test_passport):
+
     """Simple test creating a passport, test against bad values"""
```
Comment (bwrsandman) [inline passport/tests/test_passport.py:78]: Is the newline before a doctring a positive change to style?

File: passport/tests/test_passport.py [hunk 14]
```diff
@@ -26,6 +26,7 @@
 
 
 class Base_Test_passport(TransactionCase):
+
     """Simple test creating a passport
     This is a base class for passport test cases.
```
Comment (bwrsandman) [inline passport/tests/test_passport.py:30]: Is the newline before a doctring a positive change to style?

File: res_partner_affiliate/res_partner.py [hunk 15]
```diff
@@ -23,6 +23,7 @@
 
 
 class ResPartner(orm.Model):
+
     """
```
Comment (bwrsandman) [inline res_partner_affiliate/res_partner.py:26]: Is the newline before a doctring a positive change to style?

File: base_location/better_zip.py (hunk 6 above)
Comment (gurneyalex) [inline base_location/better_zip.py:26]: please no empty line between class declaration and docstring!

File: base_contact/base_contact.py (hunk 5 above)
Comment (gurneyalex) [inline base_contact/base_contact.py]: @bwrsandman it is even dangerous IMO: if someone calls read with the kwarg it will fail

and we don't want to change the import

**Comment by mdietrichc2c** [comment]
@bwrsandman for the empty lines, they were added by the script; I'll remove them.

File: base_partner_merge/base_partner_merge.py (hunk 8 above)
Comment (mdietrichc2c) [inline base_partner_merge/base_partner_merge.py]: Good point; i'll reformat it.

File: base_contact/base_contact.py (hunk 5 above)
Comment (mdietrichc2c) [inline base_contact/base_contact.py]: I wasn't sure what to do with this particular case, but it's really not a good solution. I'll revert it, even if this means we'll get a PyLint warning for now.

**Comment by bwrsandman** [comment]
@mdietrichc2c feel free to squash the commits into a single one while you're at it.

File: base_contact/base_contact.py (hunk 5 above)
Comment (gurneyalex) [inline base_contact/base_contact.py]: I submitted https://github.com/OCA/maintainer-quality-tools/issues/141  about this precise issue

**Comment by gurneyalex** [comment]
I took over the PR in #52. Closing this one.
